### PR TITLE
[FIX] web_editor: rtl table menu misalignment

### PR DESCRIPTION
**Pull request by aath-odoo**
**Description of the issue this PR addresses:**

Previously, In RTL mode, the table columns menu were misaligned, causing a visual discrepancy in the UI.Now, Implemented a fix to correctly position the table menu columns in RTL mode, ensuring proper alignment and resolving the visual misalignment issue.

task-[3695715](https://www.odoo.com/web#id=3695715&cids=2&menu_id=6478&action=4043&model=project.task&view_type=form)

### CODE REVIEW

File: addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js [hunk 1]
```diff
@@ -2715,7 +2715,9 @@ export class OdooEditor extends EventTarget {
         if (isRow && isRtl) {
             left = tableRect.right - tableUiContainerRect.x;
         } else if (isRow && !isRtl) {
-            left = elementRect.left - tableUiContainerRect.left - (isRow ? wrappedUi.clientWidth : 0);
+            left = elementRect.left - tableUiContainerRect.left - wrappedUi.clientWidth;
+        } else if (!isRow && isRtl) {
+            left = elementRect.left - tableUiContainerRect.left + wrappedUi.clientWidth;
         } else {
             left = elementRect.left - tableUiContainerRect.left - (isRow ? wrappedUi.clientWidth : 0);
         }
```
Comment (Zynton) [inline addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js:2723]: At this point we should rearrange this to be a bit clearer, don't you think?

```suggestion
        if (isRow) {
            if (isRtl) {
                left = tableRect.right - tableUiContainerRect.x;
            } else {
                left = elementRect.left - tableUiContainerRect.left - wrappedUi.clientWidth;
            }
        } else if (isRtl) {
            left = elementRect.left - tableUiContainerRect.left + wrappedUi.clientWidth;
        } else {
            left = elementRect.left - tableUiContainerRect.left - (isRow ? wrappedUi.clientWidth : 0);
        }

**Review by Zynton — COMMENTED** [review]
LGTM though I have a style suggestion just to clarify things.

File: addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js (hunk 1 above)
Comment (aath-odoo) [reply addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js:2723]: > At this point we should rearrange this to be a bit clearer, don't you think?

Sure thing :)

Comment (aath-odoo) [reply addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js]: GTG :+1:  @Zynton

**Comment by Zynton** [comment]
@robodoo r+

**Pull request closed without merging**